src/syntax/binding_in_syntax.ts

```diff
@@ -1,30 +1,44 @@
 import { BindingScopeEnum } from "../constants/literal_types";
 import { interfaces } from "../interfaces/interfaces";
 import { BindingWhenOnSyntax } from "./binding_when_on_syntax";
+import { BindingScopeScopeFactory as BindingScopeScopeFactoryInterface } from "../scope/binding-scope-scope-factory-interface";
```

Comment:
Ok to change to IBindingScopeScopeFactory ? Did not want to add to interfaces.ts internal interfaces.